pymeasure/instruments/andeenhagerling/ah2500a.py

```diff
@@ -0,0 +1,76 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from pymeasure.instruments import Instrument
+from .util import parse_reply
+
+
+class AH2500A(Instrument):
+    """ Andeen Hagerling 2500A Precision Capacitance Bridge implementation
+    """
+    id = "Andeen Hagerling AH2500A"  # device does not support '*IDN?'
```

Comment:
if this instrument does not offer SCPI capabilities, it is recommended to add `includeSCPI=False,` to the `super(..).__init__` parameters

Reply:
I decided to include `includeSCPI=False` for both instruments since even the newer one only supports a small subset of the SCPI commands implemented in `Instrument`. I therefore reimplemented the `id` property and a `reset` function. Let me know if this is ok for you.

Reply:
Sounds good.